File: elementy.py
nums = [ 1, 1, 6, 5, 5, 3, 3, 5 ]
max = (0, 0)
result = { }
for i in nums:
    result[i] = result.setdefault(i, 0) + 1
for i in result.items():
    if(i[1] > max[1]): max = i
print(max[0])

File: elementy.cpp
#include <iostream>
#include <map>
int main()
{
    int nums[] = { 1, 5, 5, 5, 4, 3 };
    std::pair<int, int> max = { 0 , 0 };
    std::map<int, int> result;
    for (int i : nums)
    {
        result[i]++;
    }
    for (std::pair<int, int> i : result)
    {
        if(i.second > max.second) 
        {
            max = i;
        }
    }
    std::cout << max.first;
}